`vscode:minitest:list` -> TypeError: no implicit conversion of nil into Integer

### Your environment

- `vscode-ruby-test-adapter` version: v0.9.0
- Ruby version: 2.7.6p219
- VS Code version: 1.69.1
- Operating System: macOS Monterey 12.4
- RSpec or Minitest version: Minitest 5.16.2

### Expected behaviour

Typically a few seconds pass and my tests load. When they don't it is usually because I have a `rake` error...e.g. my postgres container isn't running the app's database. At this point I'd start the database and perform a VS Code `Developer: Reload Window`. 

### Actual behaviour

In my case, I have two separate Rails applications who are suddenly failing to list tests even when everything seems to be working. I do have some _rake_ noise where some Ruby/Rails trace/deprecation messages are appearing but they are in no way producing a non-zero error code preventing the `vscode:minitest:list` command from succeeding.

Here are the log messages from the extension; note the **no implicit conversion of nil into Integer** ... I'm seeing this in multiple projects.

```
[2022-07-12 20:24:25.727] [INFO] Getting a list of Minitest tests in suite with the following command: DISABLE_SPRING=1 PARALLEL_WORKERS=1 ./bin/rake -R $EXT_DIR vscode:minitest:list
[2022-07-12 20:24:25.727] [INFO] Initialization finished
[2022-07-12 20:24:30.287] [ERROR] Error while finding Minitest test suite: Command failed: DISABLE_SPRING=1 PARALLEL_WORKERS=1 ./bin/rake -R $EXT_DIR vscode:minitest:list
... (I have some deprecation messages from rails here BUT nothing generating an error code) ... 
rake aborted!
TypeError: no implicit conversion of nil into Integer
/Users/.../.vscode/extensions/connorshea.vscode-ruby-test-adapter-0.9.0/ruby/vscode/minitest/tests.rb:33:in `block in build_list'
/Users/.../.vscode/extensions/connorshea.vscode-ruby-test-adapter-0.9.0/ruby/vscode/minitest/tests.rb:32:in `map'
/Users/.../.vscode/extensions/connorshea.vscode-ruby-test-adapter-0.9.0/ruby/vscode/minitest/tests.rb:32:in `build_list'
/Users/.../.vscode/extensions/connorshea.vscode-ruby-test-adapter-0.9.0/ruby/vscode/minitest/tests.rb:9:in `all'
/Users/.../.vscode/extensions/connorshea.vscode-ruby-test-adapter-0.9.0/ruby/vscode/minitest.rb:25:in `list'
/Users/.../.vscode/extensions/connorshea.vscode-ruby-test-adapter-0.9.0/ruby/vscode.rake:8:in `block (3 levels) in <main>'
Tasks: TOP => vscode:minitest:list
(See full trace by running task with --trace)
```

--- a/ruby/vscode/minitest/tests.rb
+++ b/ruby/vscode/minitest/tests.rb
@@ -28,8 +28,8 @@ module VSCode
       end
 
       def build_list
-        if Minitest.respond_to?(:seed) && Minitest.seed.nil?
-          Minitest.seed = (ENV['SEED'] || srand).to_i % 0xFFFF
+        if ::Minitest.respond_to?(:seed) && ::Minitest.seed.nil?
+          ::Minitest.seed = (ENV['SEED'] || srand).to_i % 0xFFFF
         end
 
         tests = []

--- a/ruby/vscode/minitest/tests.rb
+++ b/ruby/vscode/minitest/tests.rb
@@ -28,6 +28,10 @@ module VSCode
       end
 
       def build_list
+        if Minitest.respond_to?(:seed) && Minitest.seed.nil?
+          Minitest.seed = (ENV['SEED'] || srand).to_i % 0xFFFF
+        end
+
         tests = []
         ::Minitest::Runnable.runnables.map do |runnable|
           file_tests = runnable.runnable_methods.map do |test_name|

